packages/shared-core/tsconfig.build.json

```diff
@@ -15,7 +15,8 @@
     "skipLibCheck": true,
     "paths": {
       "@budibase/types": ["../types/src"]
-    }
+    },
+    "tsBuildInfoFile": "dist/tsconfig.tsbuildinfo"
```

Comment:
This fixes the issue with shared-core not being properly rebuilt on dev. We had the `build` and `build:dev` commands generating different tsbuildinfo, and it was conflicting with the incremental builds